config/servicemappings/run.yaml

```diff
@@ -84,6 +83,15 @@ spec:
         kind: IAMServiceAccount
         version: v1beta1
         group: iam.cnrm.cloud.google.com
+    - tfField: template.template.volumes.cloud_sql_instance.instances
+      description: |-
+        The Cloud SQL instance connection names, as can be found in https://console.cloud.google.com/sql/instances. Visit https://cloud.google.com/sql/docs/mysql/connect-run for more information on how to connect Cloud SQL and Cloud Run. Format: {project}:{location}:{instance}
+      key: instances
```

Comment:
Should be `instancesRef` per the naming convention: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.ConfigureResourceReferences.md#configure-reference-resource-in-the-service-mappings, point 3

Reply:
I think for a list-typed reference field, we should use name `instanceRefs` instead.

Reply:
Hi, I used `instances` to be coherent with the [instance field of Run Services](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/servicemappings/run.yaml#L204-L212) but I changed it to `instanceRefs` in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1688/commits/249d2177884671256729fe7d315c97559d5df31a